### DISCUSSION TITLE
Thank You Thank You, And a give Back....

### DISCUSSION THREAD

**Discussion by chuckkay** [OpenNeptune3D/OpenNept4une#57] [General]
Works Great...... What a Pleasure, And to think I was about to trash my Elegoo Neptune 4 Max, now its actually working and working well....      FYI I Have Successfullly Upgraded my N4Max to a Revo 60W Hotend with High Flow Nozzles....  It's a fairly simple upgrade and a work in progress to make simpler...  parts total $35-$50....  will post info if requested

**Comment by barrenechea** [comment]
@chuckkay, please do! Documenting what this series of printers supports and how-to's are gold for this community right now 😄

**Comment by chuckkay** [comment]
![IMG_0276](https://github.com/OpenNeptune3D/OpenNept4une/assets/10617746/78d88c65-b8f6-475a-b0d1-ffb4e78ced70)
Posted this on Elegoo Facebook group, hoping it will also raise awareness of you fantastic work as well
